src/core/boxes.c

```diff
@@ -30,6 +30,12 @@
 #include "util.h"
 #include <X11/Xutil.h>  /* Just for the definition of the various gravities */
 
+void
+sn_metarectangle_free (MetaRectangle *metarectangle)
```

Comment:
Wrong prefix... Should not it be `meta_rectangle_free`?

Reply:
You are right, I will change sn_metarectangle_free to meta_rectangle_free